calendar/calendar.yaml

```diff
@@ -193,6 +193,29 @@
       Meeting notes: https://docs.google.com/document/d/1DmMhcae081SItH19gSqBpFtPfbkr9dFhSMCgs-JKzNo/edit?usp=sharing
   organizer: tarilabs
 
+- id: kf042
+  name: Kubeflow Spark Operator Community Call
```

Comment:
For consistency with the other meetings, and to help people in other timezones know it's not required for them to attend:

 ```suggestion
   name: Kubeflow Spark Operator Meeting (US + EMEA)
```

Reply:
Since we are not planning to host meeting in other timezone in the near future, do we need to indicate that this meeting is only for US + EMEA ?
For example, Manifests WG also doesn't have this notice: https://github.com/kubeflow/community/blob/b60983b88dfe551d7ba01235fe6d287168c06d36/calendar/calendar.yaml#L124

Reply:
@andreyvelich it's still important for users, so they know the meeting is not aimed at them.

And if we have another meeting time in the future, it makes it easier to distinguish (for recordings and in the calendar config).

Reply:
We don't know schedule for all folks in the APAC timezone, maybe some of them are ok to join 10am PST meeting.
From my perspective user might get confused with this meeting name, since we don't have other timezone available:
`Kubeflow Spark Operator Meeting (APAC)`

@vara-bonthu @yuchaoran2011 thoughts ?

Reply:
It seems unlikely that someone in APAC would be able to attend. (It's at 3:00 a.m. in Sydney)

And if they were wanting to attend, It would still help them to know which time zones are most sensible for the meeting.